packages/core/src/App/Containers/Redirect/redirect.jsx

```diff
@@ -139,26 +139,57 @@ const Redirect = observer(() => {
             setResetTradingPasswordModalOpen(true);
             break;
         }
+        case 'payment_deposit': {
+            if (is_next_wallet) {
```

Comment:
We need to remove these checks, let's not make redirection depend on the feature flag. Migrated users should be redirected to wallets and non migrated users should be redirected to cashier.

Reply:
agree - in previous PRs I've seen has_wallet flag which could be used for that potentially?

Reply:
Updated 👍 I've also updated the other redirections in the component to not rely on the feature flag